app/styles/component_tree.scss

```diff
@@ -7,6 +7,18 @@
   margin: 0 3px;
   min-height: 22px;
   position: relative;
+
+  &:hover {
+    background-color: var(--base05);
+
+    .component-tree-item__action {
+      opacity: 1;
+    }
+
+    .component-tree-item__action.disabled {
+      opacity: 0.65;
+    }
+  }
```

Comment:
Remove this hunk (it was moved down due to linter complaints)

Reply:
I removed the one that was moved down. This is now the only copy of this.

Reply:
Weren't the linter unhappy with it? (Something something descending specificity, seen in the `ember build` output)